### ISSUE TITLE
Federated shares need some UX/UI improvement to be understood (see my UX suggestions in screenshots)

### ISSUE THREAD

**Issue by Jerome-Herbinet** [nextcloud/server#33135]
**Is your feature request related to a problem? Please describe.**
When entering a Federated Cloud ID in the sharing field, there is one that makes it difficult to understand the federated share option : 

There is no way to understand what is in front of your eyes (no icon, no textual information) : 

![Capture d’écran de 2022-07-06 15-06-427](https://user-images.githubusercontent.com/33763786/177564538-6a8a8d66-7f7c-47a2-a023-ac55d8c85a5b.png)

**Describe the solution you'd like**
See my suggestion ("remote" in brackets + icon) : 

![2022-07-06_15-27](https://user-images.githubusercontent.com/33763786/177563804-9d3e8449-fa49-43fb-ba39-a66120179d5c.png)

Once federated sharing is done, here is what I suggest in place of the letter in the bubble to the left of the Federated Cloud ID (icon) : 
![2022-07-06_15-26](https://user-images.githubusercontent.com/33763786/177565336-a192a440-d600-49c2-87e1-e1f5ec587b72.png)
This last suggestion is a little less important, but "while we're at it" ... I suggest it.

**Comment by Jerome-Herbinet** [comment]
@nextcloud/designers

**Comment by szaimen** [comment]
Yeah, sounds good to me to add an icon for federated users. Maybe use this one for example? https://fonts.google.com/icons?icon.style=Outlined&icon.set=Material+Symbols&icon.query=cloud&selected=Material+Symbols+Outlined:filter_drama:FILL@0;wght@400;GRAD@0;opsz@24&icon.size=24&icon.color=%23e8eaed
WDYT @nextcloud/designers ?

**Comment by marcoambrosini** [comment]
I would use something like this instead
https://pictogrammers.com/library/mdi/icon/hub/

But I would modify it a bit so that the central node is not bigger than the outer ones

**Comment by jancborchardt** [comment]
Sounds like a nice enhancement.
- For wording, let’s use "federated" instead of "remote" since we say "Federated Cloud ID" and "federation"
- As icon, the Hub icon as proposed by @marcoambrosini is indeed fitting – here it is from Material Symbols: https://fonts.google.com/icons?icon.style=Outlined&icon.set=Material+Symbols&icon.query=hub&selected=Material+Symbols+Outlined:hub:FILL@1;wght@400;GRAD@0;opsz@24&icon.size=24&icon.color=%23e8eaed

**Comment by susnux** [comment]
> Once federated sharing is done, here is what I suggest in place of the letter in the bubble to the left of the Federated Cloud ID (icon) 

This is not always the case you can have federation with known users that have proper avatars, from my point of view you should not handle federated users differently. E.g. it is a colleague from a different department, why not shot their avatar?

(Agree with the other points and with using the icon in case there is no avatar)

**Comment by jancborchardt** [comment]
@susnux true, and we already have the notice "(federated)" behind the name. It should be enough to serve as info, but we need to aim for a great experience no matter if it’s shared on the same instance or federated.

By that logic I would also say if there is no avatar we can use the regular avatar placeholders?

**Comment by Jerome-Herbinet** [comment]
Ping @jancborchardt @susnux any news ?
